src/main/java/edu/ucsb/cs156/courses/controllers/CourseOverTimeBuildingController.java

```diff
@@ -22,25 +26,19 @@ public class CourseOverTimeBuildingController {
 
   @Autowired ConvertedSectionCollection convertedSectionCollection;
 
-  @Operation(summary = "Get a list of courses over time, filtered by (abbreviated) building code")
-  @GetMapping(value = "/buildingsearch", produces = "application/json")
+  @Operation(
```

Comment:
Same thing here.  Instead, of changing the code (and breaking the contract with the frontend, please consider add a NEW method along side the old one.

Then in a two later PRs;
* change the frontend to match the new backend endpoints
* THEN remove the old backend code that's no longer needed.